addons/website_event/views/event_templates.xml

```diff
@@ -580,7 +580,7 @@
             </div>
         </div>
         <div class="col-lg-4 pt-3 pt-lg-0 pl-2 pl-lg-0">
-            <button type="submit" class="btn btn-primary o_wait_lazy_js btn-block a-submit" t-attf-id="#{event.id}">Register</button>
+            <button type="submit" class="btn btn-primary o_wait_lazy_js btn-block a-submit" onclick="event.preventDefault()" t-attf-id="#{event.id}" disabled="disabled">Register</button>
```

Comment:
Isn't this `onclick` an "overkill", as we have `.off` and the button is now first disabled?

```suggestion
            <button type="submit" class="btn btn-primary o_wait_lazy_js btn-block a-submit" t-attf-id="#{event.id}" disabled="disabled">Register</button>
```

Reply:
indeed, I added the `onclick` at the beginning then notice that it was confusing visually as the button was clickable but would do nothing, then I added the `disabled`. But I forgot that it would be redundant, thank you for the comment, i did update the code on the matter as you proposed :)